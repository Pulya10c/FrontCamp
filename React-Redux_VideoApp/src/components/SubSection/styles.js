import styled from 'styled-components';

export const SubSection = styled.section`
  display: grid;
  align-items: center;
  background-color: grey;
  min-height: 55px;
  padding: 10px;
`;
