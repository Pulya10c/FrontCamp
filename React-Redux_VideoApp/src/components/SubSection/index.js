import { SubSection } from './styles';

export default SubSection;
