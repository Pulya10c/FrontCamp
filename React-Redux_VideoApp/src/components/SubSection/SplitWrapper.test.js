import React from 'react';
import { shallow } from 'enzyme';
import SubSection from './';

describe('Component SubSection', () => {
  it('should render correctly', () => {
    expect(shallow(<SubSection />)).toMatchSnapshot();
  });
});
