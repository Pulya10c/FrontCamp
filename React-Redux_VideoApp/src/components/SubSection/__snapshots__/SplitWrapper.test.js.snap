// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component SubSection should render correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  display: grid;
  align-items: center;
  background-color: grey;
  min-height: 55px;
  padding: 10px;
",
        ],
      },
      "displayName": "styled.section",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "section",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
